### ISSUE TITLE
Duplicate fields when using multi-page/tabbed form

### ISSUE THREAD

**Issue by ThibautMollet** [kevinchappell/formBuilder#625]
### Description:
Each time a new formbuilder is added to a tabbed form, custom fields are duplicated

### Environment Details:

 - formBuilder Version: 2.5.3
 - Browser: chrome 61.0.3163.100
 - OS: W10

### Expected Behavior
Adding a tab doesn't duplicate custom fields

### Actual Behavior
When adding a tab, custom fields are duplicated

### Steps to Reproduce
Go to https://codepen.io/kevinchappell/pen/BKYPRp
Paste this in JS section :

```
$(function() {
  "use strict";
  var $fbPages = $(document.getElementById("form-builder-pages"));
  var addPageTab = document.getElementById("add-page-tab");
  var fbInstances = [];
      var fbOptions = {
        fields: [
            {
                type: 'text',
                label: 'foo'
            }
        ],
        disableFields: ['header', 'text', 'textarea', 'select', 'radio-group', 'number', 'date', 'checkbox-group', 'button', 'hidden', 'autocomplete', 'file', 'starRating', 'paragraph']
    };

  $fbPages.tabs({
    beforeActivate: function(event, ui) {
      if (ui.newPanel.selector === "#new-page") {
        return false;
      }
    }
  });

  addPageTab.onclick = function() {
    var tabCount = document.getElementById("tabs").children.length,
      tabId = "page-" + tabCount.toString(),
      $newPageTemplate = $(document.getElementById("new-page")),
      $newPage = $newPageTemplate
        .clone()
        .attr("id", tabId)
        .addClass("fb-editor"),
      $newTab = $(this).clone().removeAttr("id"),
      $tabLink = $("a", $newTab)
        .attr("href", "#" + tabId)
        .text("Page " + tabCount);

    $newPage.insertBefore($newPageTemplate);
    $newTab.insertBefore(this);
    $fbPages.tabs("refresh");
    $fbPages.tabs("option", "active", tabCount - 1);
    fbInstances.push($newPage.formBuilder(fbOptions));
  };

  fbInstances.push($(".fb-editor").formBuilder(fbOptions));

  $(document.getElementById("save-all")).click(function() {
    var allData = fbInstances.map(function(fb) {
      return fb.formData;
    });
    console.log(allData);
  });
});

```

then add one tab, then one another.

**Comment by ThibautMollet** [comment]
Any updates on that issue?

**Comment by kevinchappell** [comment]
#607 should address this issue but needs a minor update before merging

**Comment by ThibautMollet** [comment]
Hi Kevin.

Thank you for you reply but unfortunatelly it doesn't. I've already tried with this version, but I still have duplicated fields on my second and third tabs.. If I remove custom fields, everything is fine

**Comment by ThibautMollet** [comment]
I found a solution: I must create my own custom fields instead of overriding standard fields (like header or text).

**Comment by kevinchappell** [comment]
This should be fixed in 2.10.x now that registered custom fields are no longer global
